chain/events/filter/event.go

```diff
@@ -105,18 +105,9 @@ func (f *EventFilter) CollectEvents(ctx context.Context, te *TipSetEvents, rever
 				continue
 			}
 
-			entries := make([]types.EventEntry, len(ev.Entries))
-			for i, entry := range ev.Entries {
-				entries[i] = types.EventEntry{
-					Flags: entry.Flags,
-					Key:   entry.Key,
-					Value: entry.Value,
-				}
-			}
```

Comment:
I'm not actually sure what this code was supposed to be doing. I assume we didn't need to copy?

Reply:
I think originally we did some decoding here, which was removed